tests/ui/update-references.sh

```diff
@@ -26,21 +26,19 @@ while [[ "$1" != "" ]]; do
     STDOUT_NAME="${1/%.rs/.stdout}"
     FIXED_NAME="${1/%.rs/.fixed}"
     shift
-    if [ -f $BUILD_DIR/$STDOUT_NAME ] && \
-           ! (diff $BUILD_DIR/$STDOUT_NAME $MYDIR/$STDOUT_NAME >& /dev/null); then
-        echo updating $MYDIR/$STDOUT_NAME
-        cp $BUILD_DIR/$STDOUT_NAME $MYDIR/$STDOUT_NAME
+    if [[ -f "$BUILD_DIR"/"$STDOUT_NAME" ]] && \
+           ! (cmp -s -- "$BUILD_DIR"/"$STDOUT_NAME" "$MYDIR"/"$STDOUT_NAME"); then
```

Comment:
Is there a reason to choose `cmp -s` over `diff -q`?

Reply:
According to stackexchange, no: https://unix.stackexchange.com/a/153612 (at least performance wise)

Reply:
`cmp -s` is completely quiet, unlike `diff -q` that print a line to stdin confirm that files are differ.